code/clue.js

```diff
@@ -1,59 +1,548 @@
+  mrsPeacock,
+  colMustard,
+  mrsWhite,
+];
+
+// weapons array
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbel,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+];
+
+// rooms array
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballRoom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio,
+];
+
+////////////////////////////////////////////////////////////////////////////////////////
+
+// Mystery object
+const mystery = {
+  killer: null,
+  weapon: null,
+  room: null,
+};
+
+////////////////////////////////////////////////////////////////////////////////////////
+
+// Random selector (((if not using the mostLikelyRandomSelector)))
+const randomSelector = (array) => {
+  return array[Math.floor(Math.random() * array.length)];
+};
 
-const weapons = []
+////////////////////////////////////////////////////////////////////////////////////////
 
-const rooms = []
+// *** tryed to solve and undertand this problem in sandbox:
+//https://codesandbox.io/s/random-pet-with-favourite-pet-u11le *** WITH COMMENTS!
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
-const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
-}
+// *** Most-likely machine ***
+const mostLikelyRandomSelector = (items, favouriteItem) => {
+  // For each favouriteItem there will be space for, in this case, 7 more of them in an array. E.g. "7 knifes" or "7 kitchens".
+  const favouriteItemsDuplicated = new Array(7).fill(favouriteItem);
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+  // (...spread) Takes the array and spreads the items out.
+  // E.g. taking out all your nested individual russian dolls and lay them out in its own individual pieces.
+  // Explains Spread ---> https://www.samanthaming.com/tidbits/92-6-use-cases-of-spread-with-array/
+  // Merging arrays. RandomArray = random Array + favouriteItemsDuplicated (e.g all weapons + 7 knifes):
+  const randomArray = [...items, ...favouriteItemsDuplicated];
+
+  // Taking out a random index from the randomArray (items + favouriteItemsDuplicated)
+  const randomIndex = Math.floor(Math.random() * randomArray.length);
+
+  // Returning the randomIndex ---> [] from the randomArray
+  return randomArray[randomIndex];
+};
+
+////////////////////////////////////////////////////////////////////////////////////////
+
+// *** Clicking "KILLER CARD" ***
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+  // flips card
+  document.getElementById("killerCard").classList.add("flip");
```

Comment:
You took this project to the next level, why not have this animation again if you click the card multiple times?